doc/modules/changes/20211029_ekendall

```diff
@@ -0,0 +1,3 @@
+New: This PR combines the velocity_statistics and velocity_boundary_statistics postprocessor to get the parameter Mobility following Lourenco et al., 2020 G3
```

Comment:
You will need to give this file a different name because another one with the same name already exists.